### PR TITLE
Create MACDMP tape images

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
Add `-c` to the macdmp tool to create a tape image.  Take file names on the command line.

### CODE REVIEW

File: macdmp.c [hunk 1]
```diff
@@ -293,18 +397,59 @@ main (int argc, char **argv)
     case 'x':
       extract = 1;
       break;
+    case 'c':
+      input_word_format = &its_word_format;
```
Comment (atsampson) [inline macdmp.c:401]: Does this need to set the word formats again? They haven't changed since the code above...

**Review by atsampson — APPROVED** [review]
Looks good to me (one minor comment).

File: macdmp.c (hunk 1 above)
Comment (larsbrinkhoff) [reply macdmp.c:401]: The input/output formats are swapped.

Comment (atsampson) [reply macdmp.c:401]: Ahh!

**Pull request merged**